client/index.js

```diff
@@ -7,6 +7,7 @@ import { loadGetInitialProps, getURL } from '../lib/utils'
 import PageLoader from '../lib/page-loader'
```

Comment:
possible to `// @flow` this or should we do that in a later PR (i.e. PR just to flowtype more things)

Reply:
Yeah I didn't on purpose.

Reply:
👍